RibonDAO/core-api
# feat: add blocked status to person payment

# Description

- Added blocked status to person_payments
- Added a link on external_id when the payment_intent is not generated
- For now I think is good to create the payments, to watch the "card testers", but in the future we can refactor this part and don't create a payment in those cases

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- You can use those cards: https://stripe.com/docs/testing?testing-method=card-numbers and see if the payment is saved with the correct attributes
